service/src/main/java/cz/muni/fi/pv254/parsing/App.java

```diff
@@ -179,6 +265,7 @@ private RecommendationDTO parseRecommendation(JSONObject review,GameDTO game) {
             rec.setVotedUp(votedUp);
             rec.setVotesUp(votesUp);
             rec.setWeightedVoteScore(weightedVoteScore);
+            // TODO je to lepsie takto setGame, alebo radsej do hry pridat cely list recommendations ?
             rec.setGame(game);
```

Comment:
je lepsie to uukladat ako
recomendation.setgame(game) (Aktualne to takto robim)

alebo az na konci
game.setrecommendations(list of recommendations)

alebo je to jedno

Reply:
@VargaiD

Reply:
mne sa viac paci game.setrecommendations(list of recommendations) ale ked funguju obe tak je to jedno